app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="container-fluid">
+  <div class="row justify-content-center align-items-center">
```

Comment:
@akramerin why did you bother creating the form seperatly if you only call it once?